styles.css

```diff
@@ -0,0 +1,160 @@
+* {
+  font-family: poppins, sans-serif;
+  color: #172b4d;
+}
+
+.head {
+  width: 76px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 22px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  margin: 15px;
+}
+
+.h {
+  width: 327px;
+  height: 150px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding-top: 114px;
+  margin-left: 10px;
+}
+
+.info1 {
+  width: 327px;
+  height: 146px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-top: -20px;
+  margin-left: 10px;
+}
+
+.info2 {
+  display: flex;
+  width: 180px;
+  height: 24px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 10px;
+}
+
+.twitter {
+  /* position: absolute; */
+  left: 6.4%;
+  right: 88.27%;
+  top: 77.81%;
+  bottom: 18.88%;
+}
+
+.iinsta {
+  /* position: absolute; */
+  left: 16.8%;
+  right: 81.87%;
+  top: 78.85%;
+  bottom: 18.87%;
+  background: #505f79;
+}
+
+.ninsta {
+  /* position: absolute; */
+  left: 18.74%;
+  right: 77.87%;
+  top: 78.32%;
+  bottom: 18.87%;
+}
+
+.m {
+  /* position: absolute; */
+  left: 27.47%;
+  right: 67.2%;
+  top: 78.15%;
+  bottom: 19.21%;
+}
+
+.gh {
+  /* position: absolute; */
+  left: 38.13%;
+  right: 57.07%;
+  top: 77.81%;
+  bottom: 18.87%;
+}
+
+.peace {
+  /* position: absolute; */
+  left: 48%;
+  right: 48.22%;
+  top: 77.81%;
+  bottom: 18.87%;
+}
+
+.dotless {
+  list-style-type: none;
+}
+
+.left {
+  position: relative;
+  left: -30px;
+}
+
+.section1 {
+  width: 375px;
+  height: 718px;
+  background-image: url("images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.footer-items {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 20px;
+  width: 170.17px;
+}
+
+.menu-bar {
+  float: left;
+  margin-left: 223px;
+  margin-top: 15px;
+  font-size: 25px;
+  color: #6070ff;
+}
+
+.hdiv {
+  display: flex;
+  flex-direction: row;
+}
```

Comment:
- [ Required ] Please consider adding a `CSS` rule of `justify-content: space-between` to the `.hdiv` element to make the header section of your webpage responsive across all devices. Once this is done your `CSS` should be as follows.
```CSS
.hdiv {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
}
```